src/v3/src/components/Link/Link.tsx

```diff
@@ -37,11 +41,13 @@ const Link: UISchemaElementComponent<{
   const onClick: ClickHandler = async (e) => {
     e.preventDefault();
 
-    onSubmitHandler({
-      params: actionParams,
-      isActionStep,
-      step,
-    });
+    if (!loading) {
+      onSubmitHandler({
+        params: actionParams,
+        isActionStep,
+        step,
+      });
+    }
```

Comment:
nit for improved readability (but this is a personal preference, let others chime in if they agree or not):
```suggestion
    if (loading) {
      return;
    }
    onSubmitHandler({
        params: actionParams,
        isActionStep,
        step,
      });
```